pom.xml

```diff
@@ -1347,6 +1319,13 @@
         <version>2.10.1</version>
       </dependency>
 
+      <!-- Quality control dependencies -->
+      <dependency>
```

Comment:
Is this change actually needed for Units upgrade? I don't see the spotbugs annotations being used in the diff.

Reply:
Ah, I see there are lots of other unrelated changes in the diff... the checklist contains this for supported modules:

"The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it."

For this one we can close an eye, but please remember about it next time.

Reply:
Will do, sorry about that.

Reply:
I removed it, opened separate pull request.